packages/dds/tree/src/feature-libraries/forest-proxy/README.md

```diff
@@ -0,0 +1,7 @@
+# Forest Proxy
+
+Implementation of Forest proxy.
```

Comment:
This sentence doesn't help much. We want to answer, "what is Forest Proxy?"

From the name, I's expect it's an implementation of the forest interfaces done by wrapping a different forest using proxies. I don't think that's what it is.

At the high level, I think what this module is trying to do implement an API for reading and editing trees which are stored in an IEditiableForest that is friendly for schema aware code (aka statically typed with respect to the schema) to use. I have been calling this abstraction `EditableTree`.

I can see two ways we can organize the code for this:
1. A single module/folder that defines the both the abstraction and a single implementation. In this case I think calling it editable-tree makes sense. Proxies are just an implementation detail of that implementation.
2. Two modules. One which defines the abstraction (editable-tree) and a separate one which provides a proxy+forest based implementation of editable-tree, which I would call editable-tree-forest-proxy.

I think the use of proxies is really an implementation detail that shouldn't be in the name.